content/shmem_for.tex

```diff
@@ -0,0 +1,45 @@
+\apisummary{
+  Atomically perform a fetching bitwise inclusive or operation on a
+  remote data object.
+}
+
+\begin{apidefinition}
+
+\begin{C11synopsis}
+TYPE shmem_for(TYPE *dest, TYPE value, int pe);
```

Comment:
I prefer the new names.  Unless you think there will be a procedural advantage to splitting these up, I prefer keeping them together.  I think we will want to have both sets of changes in the spec for the next release, so it makes sense to me that they stay together.